package.json

```diff
@@ -28,6 +28,7 @@
   },
   "homepage": "https://github.com/brentvatne/react-native-scrollable-tab-view#readme",
   "dependencies": {
+    "react-static-container": "^1.0.1",
```

Comment:
I think it shipped with react-native try like here https://github.com/facebook/react-native/blob/50c2467905815dbb434beecd0e649fb4a7bc2e20/Libraries/Components/TabBarIOS/TabBarItemIOS.ios.js#L16